third_party/grpc-rust/grpc-compiler/src/codegen.rs

```diff
@@ -172,13 +172,14 @@ impl<'a> MethodGen<'a> {
         if let Some(sig) = self.client_sig() {
             w.pub_fn(&sig, |w| {
                 w.write_line(format!(
-                    "oak::grpc::{}(\"{}\", req, &self.0.invocation_sender)",
+                    "oak::grpc::{}(\"{}\", req, Some(\"type.googleapis.com/{}\"), &self.0.invocation_sender)",
```

Comment:
I got this from the existing code and from the [proto3 language guide](https://developers.google.com/protocol-buffers/docs/proto3#any), which says that "The default type URL for a given message type is `type.googleapis.com/packagename.messagename`", so the docs seem a bit inconsistent.  (The leading dot is easy to get trim, so I've done that.)

Worth an issue to track the concern about interop, or just worry about it if/when we encounter it?

Reply:
Thinking about it a bit more, it is probably OK to wait until we need it. It should only be an issue if external code is trying to unpack protos from an Any field, and they don't have the associated proto definitions. I don't think we will encounter this situation in the near future.

Reply:
In my experience, the URL resolution is basically never used, apart from perhaps "generic" command line tools used for debugging, since they deal with completely unknown types. So it's probably fine to ignore.